src/components/Garden/Stake/SideBar.tsx

```diff
@@ -7,8 +7,18 @@ import { formatTokenAmount } from '@utils/token-utils'
 
 import coin from './assets/coin.svg'
 import wallet from './assets/wallet.svg'
+import { TokenType } from '@/types/app'
 
-function Sidebar({ stakeActions, staking, token, onDepositOrWithdraw }) {
+type SidebarType = {
+  token: TokenType
+} & SideBarProps
+
+function Sidebar({
```

Comment:
Maybe camelCase here too?
```suggestion
function SideBar({
```

I saw we have another component called Sidebar.js we should get better names to it? Idk what it does to suggest good name

Reply:
I don't see the need to change this into CamelCase, in my opinion.

Reply:
Oh you want to make the component the same as the name of the file? Its makes total sense actually.

Reply:
Changed!